packages/vite-plugin/src/node/defineClientValues.ts

```diff
@@ -30,6 +30,8 @@ export function defineClientValues(code: string, config: ResolvedConfig) {
     .replace(`__MODE__`, JSON.stringify(config.mode))
     .replace(`__BASE__`, JSON.stringify(config.base))
     .replace(`__DEFINES__`, serializeDefine(config.define || {}))
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    .replace(`__HMR_TOKEN__`, JSON.stringify((config as any).webSocketToken))
```

Comment:
Opposed to making a minimal interface? 

something like

```
interface ViteConfigWithHMR extends ResolvedConfig {
  webSocketToken?: string;
}
```

which would be used like this `.replace(`__HMR_TOKEN__`, JSON.stringify((config as ViteConfigWithHMR).webSocketToken || ''))
`

Reply:
Yes. But the minimal interface extending `ResolvedConfig` seems okay too - changed in https://github.com/crxjs/chrome-extension-tools/pull/999/commits/b93dcd173e825b6116830a1e688f59fbf6dfac19